### ISSUE TITLE
Photos submitting the caption as the subject

### ISSUE THREAD

**Issue by Gregliest** [AmericanWhitewater/aw-ios#272]
This [line](https://github.com/AmericanWhitewater/aw-ios/blob/master/American%20Whitewater/API/GraphQL/AWGQLApiHelper.swift#L213) looks like it might be causing this [issue](https://github.com/AmericanWhitewater/wh2o/issues/2321).

**Comment by AWKevin** [comment]
Sure seems like it!

**Comment by yeahphil** [comment]
Hey just took a look, seems like that would definitely cause the issue.

What should go in that field though? The upload form doesn't have a subject field currently.

**Comment by AWKevin** [comment]
I can't see the API endpoints so can't tell you. @ryangroth5? 

I'd say adding the Subject field to the app would be good.

**Comment by ryangroth5** [comment]
Yeah, there are some defaults that get stuffed in there. A blank subject
field will null cause it to not pre-populate the subject.

Ryan

**Comment by ryangroth5** [comment]
when I say blank I specifically mean "" vs. null or something similar.

Ryan
